packages/mobile/src/components/details-tile/DetailsTileHasAccess.tsx

```diff
@@ -63,31 +75,176 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
   checkIcon: {
     width: spacing(6),
     height: spacing(6)
+  },
+  collectionName: {
+    color: palette.secondary
   }
 }))
 
-const DetailsTileUnlockedSection = ({
-  premiumConditions
-}: {
-  premiumConditions: PremiumConditions
-}) => {
+type HasAccessProps = {
+  renderDescription: () => ReactNode
+  nftCollection?: Nullable<
+    PremiumConditionsEthNFTCollection | PremiumConditionsSolNFTCollection
+  >
+  followee?: Nullable<User>
+  tippedUser?: Nullable<User>
```

Comment:
nit: a bit awks to have something that's both nullable & possible undefined, do we need both?